project/ws/viewer/src/lib/plugins/practice/components/overview/overview.component.ts

```diff
@@ -24,9 +25,11 @@ export class OverviewComponent implements OnInit {
     { icon: 'info', text: 'Assessment will have time duration' },
     { icon: 'info', text: 'Skipped question can be attempted again before submitting' },
   ]
+  isretakeAllowed : boolean = false;
```

Comment:
Why is it coming from environment, it should be coming from json in client-assets